app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    part = command.split()
+    if len(part) != 3 or part[0] != "cp":
+        return
+    source = part[1]
+    destination = part[2]
+    if source == destination:
+        return
+    try:
+        with open(source, "r") as old_name, open(destination, "w") as new_file:
```

Comment:
since it's not a name but opened file object
```suggestion
        with open(source, "r") as old_file, open(destination, "w") as new_file:
```